test/test_notebooks.py

```diff
@@ -1,16 +1,49 @@
+import nbformat as nbf
 import papermill as pm
 import scrapbook as sb
 import pytest
 
 
+def append_scrapbook_commands(input_nb_path, output_nb_path, scrap_specs):
+    notebook = nbf.read(input_nb_path, as_version=nbf.NO_CONVERT)
+
+    scrapbook_cells = []
+    # Always need to import nteract-scrapbook
+    scrapbook_cells.append(nbf.v4.new_code_cell(source="import scrapbook as sb"))
+
+    # Create a cell to store each key and value in the scrapbook
+    for k, v in scrap_specs.items():
+        source = "sb.glue(\"{0}\", {1})".format(k, v)
+        scrapbook_cells.append(nbf.v4.new_code_cell(source=source))
```

Comment:
This is adding the scrapbook cell at the very end of the notebook?

Reply:
yep, it's adding an additional cell and creating a new notebook